packages/amplify-e2e-core/src/categories/auth.ts

```diff
@@ -916,3 +916,81 @@ export function addAuthWithMaxOptions(cwd: string, settings: any) {
       });
   });
 }
+
+//add default auth
+export function addAuth(projectDir: string) {
+  return new Promise((resolve, reject) => {
+    spawn(getCLIPath(), ['add', 'auth'], { cwd: projectDir, stripColors: true })
+      .wait('Do you want to use the default authentication and security configuration')
+      .sendCarriageReturn()
+      .wait('How do you want users to be able to sign in')
+      .sendCarriageReturn()
+      .wait('Do you want to configure advanced settings')
+      .sendCarriageReturn()
+      .wait('"amplify publish" will build all your local backend and frontend resources')
+      .run((err: Error) => {
+        if (!err) {
+          resolve();
+        } else {
+          reject(err);
+        }
+      });
+  });
+}
```

Comment:
Duplicate of this https://github.com/aws-amplify/amplify-cli/blob/4322a326df8c5c0a89de5f5e8f46bcfd4e1ad770/packages/amplify-e2e-core/src/categories/auth.ts#L3-L21